api/v1alpha1/cruisecontroloperation_types.go

```diff
@@ -0,0 +1,80 @@
+// Copyright © 2019 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/banzaicloud/koperator/api/v1beta1"
+)
+
+//+kubebuilder:object:root=true
+//+kubebuilder:subresource:status
+
+// CruiseControlOperation is the Schema for the cruiseControlOperation API.
+type CruiseControlOperation struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec   CruiseControlOperationSpec   `json:"spec,omitempty"`
+	Status CruiseControlOperationStatus `json:"status,omitempty"`
+}
+
+//+kubebuilder:object:root=true
+// CruiseControlOperationList contains a list of CruiseControlOperation.
+type CruiseControlOperationList struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata,omitempty"`
+	Items           []CruiseControlOperation `json:"items"`
+}
+
+// CruiseControlOperationSpec defines the desired state of CruiseControlOperation.
+type CruiseControlOperationSpec struct {
+	// FailurePolicy defines how failed downscale operations should be handled.
```

Comment:
**Required**: The comment calls the field `FailurePolicy`, but the actual field is `ErrorPolicy`, this taints the manifest generation as well causing discrepancy between the field name and it's description/documentation which could confuse the user.

```suggestion
	// ErrorPolicy defines how failed downscale operations should be handled.
```